### ISSUE TITLE
“Updates are not permitted" on Eclipse 2020-12

### ISSUE THREAD

**Issue by wannestas** [fsc4j/fsc4j#26]
The installation instructions do not seem to work anymore.
Using Eclipse Version 2020-12 (4.18.0) I am unable to install FSC4J using the recommended "Install New Software" method.
On a fresh installation of Eclipse, after I type in the information in the "Install New Software" window and pressing Next, I get the following error: "Eclipse Java Development Tools will be ignored because it is already installed, and updates are not permitted.".
The only way I was able to get FSC4J working was by manually downloading the Development Tools jar from the this repository's Releases page and replacing the normal jar in my eclipse installation's Plugin folder.

**Comment by btj** [comment]
Could you please specify
- your platform (Windows, macOS, Linux)
- whether you installed Eclipse using the installer or by extracting the archive
- Whether the user account you use to run Eclipse has write access to the Eclipse installation

Doing a fresh Eclipse install and installing FSC4J into it the prescribed way still works fine on my mac. I suspect you do not have write access to your Eclipse installation. Solution: install Eclipse into a folder to which you have write access.

If you confirm that this is the problem, I will update the installation instructions.

It is funny, though, that this issue has not been reported before. If this is a problem now, why was it not a problem before? Is there something unusual about your setup, or did you make non-standard choices during installation?

**Comment by TIboStans** [comment]
On MacOS 11.3.1 and Windows 10 I get the same error as @wannestas with the latest version of Eclipse. If I install an older version of Eclipse it installs, but the checker doesn't work. It doesn't check any of the docs written (even with the -ea as VM argument).

**Comment by btj** [comment]
@TIboStans FSC4J is known not to work with Eclipse 2021-03. Your issue when using 2020-12 sounds different from that of @wannestas . Please open a separate issue for it.

**Comment by wannestas** [comment]
I am running Arch Linux and installed `eclipse-java` via my package manager (which extracts from the archive as far as I can tell), but as you said it seemed to be an ownership issue. After using `chown` to change the eclipse folder's owner, FSC4J installed correctly and now works as intended.

I only report this now because I only recently ran into the issue. In february the installation worked without issue, but I recently switched to arch, after which I had this issue.
